packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -1,6 +1,5 @@
 /* eslint-disable class-methods-use-this */
-import isEqual from 'lodash.isequal';
-import { isEmpty } from 'ramda';
+import { isEmpty, equals } from 'ramda';
```

Comment:
Should we use the common utility `!isNotEmpty` here as it seems to handle things correctly?